nibabel/parrec.py

```diff
@@ -610,6 +610,7 @@ def __getitem__(self, slicer):
         # Broadcast scaling to shape of original data
         slopes, inters = self._slice_scaling
         fake_data = strided_scalar(self._shape)
+        fake_data.flags.writeable = False
```

Comment:
How about

```
# Create broadcast array using writeable memory for numpy 1.10
fake_data = strided_scalar(np.array(self._shape))
```

?  Only because the writeable line looks a bit frightening.